solidity/contracts/converter/BancorConverter.sol

```diff
@@ -986,15 +1013,28 @@ contract BancorConverter is IBancorConverter, TokenHandler, SmartTokenController
         return y;
     }
 
+    /**
+      * @dev calculates the nearest integer to a given quotient

```

Comment:
I think this function should be in the bancor formula contract, seems like generic math?

Reply:
@yudilevi : The thing is, it's not related to BancorFormula in any way.
We do have some generic functions in BancorFormula (like `power` and `floorLog` and so on).
But those are used internally by the formula functions, and are therefore not exposed to external users.
We have a total of 3 such general math functions in BancorConverter:
- `decimalLength`
- `roundDiv`
- `geometricMean`
We could place them in a generic Math library/contract, or some Util library/contract.
But none of those should be implemented and exposed in BancorFormula IMO.

Reply:
I'm planning to rename BancorFormula to Math in the future

Reply:
@yudilevi:
I suggest in that case to put these functions in a library, and possibly also do that with the general math functions in BancorFormula.
The advantage of using a library is that it will not yield external function calls AFAIK.
I'm not quite sure how to link a library upon deployment and all, but I'm sure we can find out.
In either case, not sure it critical to handle this ATM.
On the other hand, using a contract might help in reducing the bytecode size of the BancorConverter contract (though I'm not sure how much of that we can get from moving those 3 functions to a separate contract).

Reply:
I don't want to deal with libraries specifically because of the deployment overhead.
And yes - the reason I'm bringing it up now is because I want to keep all the error messages and we'll need to get rid of some of them if we can't get the binary size smaller.